docs/egress.md

```diff
@@ -127,6 +131,17 @@ The `externalIPPool` field specifies the name of the `ExternalIPPool` that the
 be assigned to. It can be empty, which means users should assign the `egressIP`
 to one Node manually.
 
+### RateLimit
```

Comment:
From the lessons of FQDN, I think we need an alpha feature gate for this support of this feature, otherwise user would misunderstand it shares the maturity of Egress feature.

Reply:
Added a toggle.

Reply:
A config toggle is different from a feature gate. Also, you are setting the toggle to true by default, which defeats a bit the purpose of "protecting" users from an experimental feature.

I think Quan is suggesting a dedicated feature gate here, and I tend to agree with him. The toggle IMO may not be necessary since the feature is API driven and users should just be able to omit the `rateLimit` field altogether for all their Egress resources if they don't want to use the feature.

At least it feels like we should start with just the feature gate. When we graduate the feature gate, we could consider introducing the toggle, but I am not convinced it will be necessary.

Reply:
I'm thinking exactly the same as Antonin. No config toggle, just an alpha feature gate, like `EgressTrafficShaping`

Reply:
Ah I see. Let me change it.

Reply:
Done.